.github/workflows/build-manywheel-images.yml

```diff
@@ -1,6 +1,9 @@
 name: Build manywheel docker images
 
 on:
+  push:
```

Comment:
do you need to specify `paths` here too?
```suggestion
  push:
    paths:
      - manywheel/Dockerfile
```

Reply:
oh good call